k8s/templates/prober-deployment.yaml

```diff
@@ -31,7 +31,10 @@ spec:
         - name: PROBER_STUDIO_PRODUCTION_MODE_ON
           value: "yes"
         - name: PROBER_STUDIO_USERNAME
-          value: lingyi+prober@learningequality.org
+          value: channeladmin@learningequality.org
```

Comment:
Hmm should this be exposed like this? This might make this email accessible by spammers. How can we make this hidden?

Reply:
ah I see. makes sense to me! I can make this as a secret just like the password